js/apps/admin-ui/src/dashboard/Dashboard.tsx

```diff
@@ -67,48 +69,47 @@ const EmptyDashboard = () => {
   );
 };
 
+type FeatureItemProps = {
+  feature: FeatureRepresentation;
+};
+
+const FeatureItem = ({ feature }: FeatureItemProps) => {
+  const { t } = useTranslation("dashboard");
```

Comment:
I think we can omit the namespace here now right?
```suggestion
  const { t } = useTranslation();
```